pkg/smokescreen/acl_loader_v1_test.go

```diff
@@ -190,7 +190,7 @@ func TestLoadFromYaml(t *testing.T) {
 		a.Equal(0, len(acl.GlobalAllowList))
 	}
 
-    // Load a sane config with global lists
+	// Load a sane config with global lists
```

Comment:
We should add tests in this file to ensure the correct reason string is supplied for each case.
- Blocked because it matches an ip rule
- Blocked due to enforce mode
- Blocked due to global deny list
- Allowed due to `allowed_domain` presence
- Allowed due to `open` or `report` mode
- Allowed due to `global_allow_list`

Reply:
Thanks will do. Wanted to throw this up to see if I had the right idea.

Reply:
I'll add some more tests for ip rules.

Reply:
Eh, actually. Not sure if there's an easy way to mock this.